src/server/api/routers/group.ts

```diff
@@ -9,6 +9,12 @@ import {
 export const groupRouter = createTRPCRouter({
   adminGetGroupList: adminProcedure.query(async ({ ctx }) => {
     // TODO: isi logic disini
+    const groups = await ctx.prisma.group.findMany({
+      select: {
+        id: true,
```

Comment:
select nama groupnya juga ya sesuai di database
```suggestion
        id: true,
        group: true
```